src/course-checklist/ChecklistSection/ChecklistItemComment.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage, FormattedNumber } from '@edx/frontend-platform/i18n';
+import { Hyperlink, Icon } from '@openedx/paragon';
+import { ModeComment } from '@openedx/paragon/icons';
+import messages from './messages';
+
+const ChecklistItemComment = ({
+  checkId,
+  outlineUrl,
+  data,
+}) => {
+  const commentWrapper = (comment) => (
+    <div className="row m-0 mt-4 border-top no-gutters align-items-center" data-identifier="comment">
+      <div className="col-1">
+        <Icon src={ModeComment} size="lg" />
+      </div>
+      <div className="col my-4 small">
+        {comment}
+      </div>
+    </div>
+  );
+
+  if (checkId === 'gradingPolicy') {
```

Comment:
It's not my place to say how much testing should be done here, since this is in part just copying over things. I'll just make a very rough recommendation that you can choose if and how to follow.

I always make a distinction between business logic and visual logic. Business logic is important to test, while visual logic is really less important until someone complains. So in this part, for example, grading values are calculated. I would say that is business logic.

Business logic is often very easy to test because you often can just extract logic into a function without side effects and check if the return is correct. In this case for example you could extract the calculation for grading policy. That is not relevant to make sure the code works now, instead, why it could be valuable is that we prevent introducing bugs later when we change things.

I'll just point out further places where I see untested business logic and you can decide if you want to add any tests.